packages/block-template/templates/custom-element/src/dev.tsx

```diff
@@ -3,7 +3,6 @@
  * This file is not bundled with the block during the build process.
  */
 import { MockBlockDock } from "mock-block-dock";
-import React from "react";
 import ReactDOM from "react-dom";
```

Comment:
We probably ought to apply the same to `react-dom`, right?

Reply:
Makes sense! Done: https://github.com/blockprotocol/blockprotocol/pull/441/commits/035d1e952ec284297aea8d2ff098c549679aca02